tools/maven/checkstyle.xml

```diff
@@ -120,6 +120,11 @@ This file is based on the checkstyle file of Apache Beam.
             <message key="import.illegal" value="{0}; Use amoro shaded guava instead."/>
         </module>
 
+        <module name="IllegalImport">
```

Comment:
Not sure if we need to update this with `Regex` property on?

https://checkstyle.sourceforge.io/checks/imports/illegalimport.html#Properties